test/rs/src/bin/test_client.rs

```diff
@@ -492,8 +483,8 @@ fn make_thrift_calls(
         s_cmp_nested_2.insert(Numberz::SIX, empty_insanity);
 
         let mut s_cmp: BTreeMap<UserId, BTreeMap<Numberz, Insanity>> = BTreeMap::new();
-        s_cmp.insert(1 as UserId, s_cmp_nested_1);
-        s_cmp.insert(2 as UserId, s_cmp_nested_2);
+        s_cmp.insert(1_i64, s_cmp_nested_1);
```

Comment:
Could you leave these two as before please?

Reply:
`1 as UserId` will cause clippy::unnecessary_cast

```
error: casting integer literal to `i64` is unnecessary
```